less/variables.less

```diff
@@ -11,4 +11,10 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+// mobile
+@mobile: ~"(max-width: 500px)";
+
```

Comment:
- Use escaping to create a variable named @mobile that contains this value: (max-width: 500px). Use the variables.less file to house your styling. - DONE! 

- Match the design file at 500px as well as you can - DONE!